### PR TITLE
protrekkr: Updated SOURCE_URI

### PR DESCRIPTION
**Pull request by kenmays**
Closes: https://github.com/haikuports/haikuports/issues/4754

### CODE REVIEW

**Comment by Begasus** [comment]
At least DESCRIPTION needs to be re-written to follow the guidelines (no new line after DESCRIPTION=")
Has this been checked on both arch's?

**Comment by kenmays** [comment]
Tested on R1B1.

**Comment by Begasus** [comment]
> Tested on R1B1.

64bit and 32bit?

**Comment by kenmays** [comment]
32-bit x86 tested per screenshot.

**Comment by Begasus** [comment]
As @waddlesplash added the type-bug in the original issue maybe he's the best to judge this :)

File: media-sound/protrekkr/protrekkr-2.5.4.recipe [hunk 1]
```diff
@@ -1,6 +1,5 @@
 SUMMARY="Tracker program to create small sized electronic music"
-DESCRIPTION="
-ProTrekkr (formerly known as NoiseTrekker by Juan Antonio Arguelles Rius aka \
+DESCRIPTION="ProTrekkr (formerly known as NoiseTrekker by Juan Antonio Arguelles Rius aka \
```
Comment (Begasus) [inline media-sound/protrekkr/protrekkr-2.5.4.recipe:2]: Keep the 80char limit as in the guidelines

File: media-sound/protrekkr/protrekkr-2.5.4.recipe [hunk 2]
```diff
@@ -9,11 +8,11 @@ HOMEPAGE="https://github.com/falkTX/protrekkr/"
 COPYRIGHT="2008-2010 Franck Charlet.
 	Based on Juan Antonio Arguelles Rius's NoiseTrekker."
 LICENSE="BSD (3-clause)"
-REVISION="2"
-SOURCE_URI="svn+http://protrekkr.googlecode.com/svn/trunk/#670"
+REVISION="1"
+SOURCE_URI="https://github.com/falkTX/protrekkr/archive/0b96ba56379eb179423e9dcbec31a08bf0326c9f.tar.gz"
```
Comment (Begasus) [inline media-sound/protrekkr/protrekkr-2.5.4.recipe:12]: Missing checksum?

Comment (Begasus) [inline media-sound/protrekkr/protrekkr-2.5.4.recipe:12]: Use srcGitRev for a git checkout, add SOURCE_DIR ...

**Comment by Begasus** [comment]
At this stage it can't be build with haikuporter ....

**Comment by Begasus** [comment]
Got it to build but crashes on launch, lots of warnings (and one error related to -Wnarrowing)

**Comment by kenmays** [comment]
@Begasus - Was using the original patchset/recipe here versus my newer set.  Retested build successfully on hrev54115 x86. Retested previous build and it works without crashing during audio playback. Feel free to rebuild and test with newer patchset/recipe..

**Comment by Begasus** [comment]
OK, narrowed it down, launches from Terminal and Tracker, crashes when launched from Deskbar

**Comment by Begasus** [comment]
There are a lot of {} which aren't needed (for instance ... protrekkr${secondaryArchSuffix})
Did you get a chance to see if it also crashes there when launched from Deskbar?

**Comment by kenmays** [comment]
Deskbar issue? I run it from standard default GUI menu. No issues or crashes after full rebuild and retest on hrev54215 x86.

**Comment by Begasus** [comment]
Maybe disable addAppDeskbarSymlink in the recipe and leave the issue https://github.com/haikuports/haikuports/issues/4990 open untill there is a fix for launching it from Deskbar?

**Pull request merged**